source/code/go/src/plugins/oms.go

```diff
@@ -734,9 +741,22 @@ func PostDataHelper(tailPluginRecords []map[interface{}]interface{}) int {
 
 		stringMap := make(map[string]string)
 
-		stringMap["LogEntry"] = ToString(record["log"])
+		logEntry := ToString(record["log"])
+		logEntryTimeStamp := ToString(record["time"])
+		if isDockerContainerRuntimeEngine == false {
```

Comment:
Please check for 'non-docker' and switch/if as appropriate to minimize impact. Leave/assume that default runtime is docker/moby. So this way, if your container runtime detection logic fails, only containerd clusters (which are very low count as of now) are impacted and not all of the clusters we monitor.

Reply:
removed this piece of code since not required any more.